tests/e2e/testcases/localnetwork/suite.go

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package localnetwork
+
+import (
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+var _ = ginkgo.Describe("[Local Network]", ginkgo.Ordered, func() {
+	ginkgo.AfterEach(func() {
+		commands.CleanNetwork()
+	})
+
+	ginkgo.It("can get correct status of local network", func() {
+		// no network started
+		out, err := commands.GetNetworkStatus()
+		gomega.Expect(err).ShouldNot(gomega.BeNil())
+		gomega.Expect(out).Should(gomega.ContainSubstring("network is not running"))
+
+		// start local network
+		out = commands.StartNetwork()
+		gomega.Expect(out).Should(gomega.ContainSubstring("Network ready to use"))
+
+		// check network status after starting
+		out, err = commands.GetNetworkStatus()
+		gomega.Expect(err).Should(gomega.BeNil())
+		gomega.Expect(out).Should(gomega.ContainSubstring("Network is Up"))
+
+		// Stop the network
+		commands.StopNetwork()
+
+		// check network status after stopping
+		out, err = commands.GetNetworkStatus()
+		gomega.Expect(err).ShouldNot(gomega.BeNil())
+		gomega.Expect(out).Should(gomega.ContainSubstring("network is not running"))
+	})
+
+	ginkgo.It("can start then stop a started network", func() {
+		out := commands.StartNetwork()
+		gomega.Expect(out).Should(gomega.ContainSubstring("Network ready to use"))
+
+		commands.StopNetwork()
```

Comment:
we need to be able to check command err return, and force errors conditions in this testsuite

Reply:
addressed.